x/tx/go.mod

```diff
@@ -1,4 +1,4 @@
-module cosmossdk.io/tx
+module cosmossdk.io/x/tx
```

Comment:
A new vanity url must be created here: https://github.com/cosmos/vanity

Reply:
https://github.com/cosmos/vanity/pull/32